common/api/core-common.api.md

```diff
@@ -331,6 +331,75 @@ export namespace AreaPattern {
     }
 }
 
+// @public
+export namespace AtmosphericScattering {
+    export interface Props {
+        atmosphereHeightAboveEarth?: number;
+        brightnessAdaptationStrength?: number;
+        densityFalloff?: number;
+        inScatteringIntensity?: number;
+        minDensityHeightBelowEarth?: number;
+        numInScatteringPoints?: number;
+        numOpticalDepthPoints?: number;
+        outScatteringIntensity?: number;
+        scatteringStrength?: number;
+        wavelengths?: WavelengthsProps;
+    }
+    export class Settings implements Props {
+        // (undocumented)

```

Comment:
Move documentation from `Props` to `Settings`; documentation on each property `xyz` of `Props` should simply say `See [[Settings.xyz]]`.